src/Workleap.DomainEventPropagation.Abstractions/DomainEventAttribute.cs

```diff
@@ -20,10 +16,10 @@ public DomainEventAttribute(string name, EventSchema eventSchema = EventSchema.E
         }
 
         this.Name = name;
-        this.Schema = eventSchema;
+        this.EventSchema = eventSchema;
     }
 
     public string Name { get; }
 
-    public EventSchema Schema { get; }
+    public EventSchema EventSchema { get; }
```

Comment:
A diagnostic was reported, but it was solely because the name of the property didn't match the constructor argument.